packages/local-build-plugin/src/ios.ts

```diff
@@ -23,17 +23,20 @@ export async function buildIosAsync(
     logger,
     logBuffer,
     uploadArtifact: async ({ artifact, logger }) => {
-      if (artifact.type === ManagedArtifactType.APPLICATION_ARCHIVE) {
-        return await prepareArtifacts(artifact.paths, logger);
-      } else if (artifact.type === ManagedArtifactType.BUILD_ARTIFACTS) {
-        return await prepareArtifacts(artifact.paths, logger);
+      if (isManagedArtifact(artifact)) {
+        return await prepareArtifacts({
+          artifactPaths: artifact.paths,
+          logger,
+          artifactType: artifact.type,
+        });
       } else {
         return null;
       }
     },
     env,
     metadata,
     skipNativeBuild: config.skipNativeBuild,
+    shouldUploadXcodeBuildLogsOnSuccess: false,
```

Comment:
Instead of adding a flag to context, can we handle special-case this in `uploadArtifact` a couple lines above? Or, can we just always "upload" Xcode build logs?